srml/support/procedural/src/storage/impls.rs

```diff
@@ -571,8 +604,11 @@ impl<'a, I: Iterator<Item=syn::Meta>> Impls<'a, I> {
 				for #name<#trait_and_instance> #where_clause
 			{
 				type Query = #value_type;
-
 				type Hasher = #scrate::#hasher;
+				// NOTE: this must not be used as (). If a method wants to use this, A proper value
+				// (similar to normal map) should be passed in to get consistent behavior with other
+				// Defaults.
+				type Default = ();
```

Comment:
hmm yes ok, maybe it worth implementing it as well even if we indeed don't use it.

Reply:
Fair point: having a bit of extra code is better than having ourselves confused in 6 months time not knowing what is wrong.

Reply:
Linked map now has the same decode_length 👍